.travis.yml

```diff
@@ -28,7 +31,7 @@ services:
   - redis-server
 
 before_install:
-  - if [[ $TRAVIS_PHP_VERSION != 7.1 ]] ; then phpenv config-rm xdebug.ini; fi
+  - if [[ $TRAVIS_PHP_VERSION != 7.2 ]] ; then phpenv config-rm xdebug.ini; fi
```

Comment:
This line basic disables xdebug on laravel tests, when 7.1 was added to the travis file xdebug were'nt supporting it, later it started supporting 7.1 and [we removed this if statement on master branch](#19211).

The only thing is that 7.1 tests will run faster after this change.